src/devtoolsPanel.ts

```diff
@@ -266,7 +266,7 @@ export class DevToolsPanel {
                 <script src="${scriptUri}"></script>
             </head>
             <body>
-                <iframe id="host" frameBorder="0" src="${htmlUri}"></iframe>
+                <iframe id="host" frameBorder="0" src="${htmlUri}?ws=trueD&experiments=true&edgeThemes=true"></iframe>

```

Comment:
why is this needed?

Reply:
This used to be in the devtools.html file where it linked inspector.html as an iframe. Since that file no longer exists we need to move the query params to where inspector.html is now loaded, which is here.